src/commands/farm.rs

```diff
@@ -2,23 +2,29 @@ use std::io::Write;
 use std::sync::atomic::{AtomicBool, Ordering};
 use std::sync::Arc;
 
-use color_eyre::eyre::{eyre, Context, Result};
+use color_eyre::eyre::{eyre, Context, Error, Result};
 use futures::prelude::*;
 use indicatif::{ProgressBar, ProgressStyle};
 use owo_colors::OwoColorize;
 use single_instance::SingleInstance;
+use sp_core::H256;
```

Comment:
I think you don't need it. You should use `subspace_sdk::node::Hash` type alias.